### DISCUSSION TITLE
Is it possible to import data (e.g. - vectors in parquet files) into milvus standalone with local storageType?

### DISCUSSION THREAD

**Discussion by liorf95** [milvus-io/milvus#36439] [Q&A and General discussion]
According to https://milvus.io/docs/import-data.md a minio remote instance is required - but is it possible also to import into milvus standalone with local storageType?

**Comment by yhmo** [comment]
I just tested import with local storage type, it could not work. I think the current code logic doesn't support local storage.

**Comment by xiaofan-luan** [comment]
I think this is doable but need to be implemented. 
Maybe just some bug to fix? please open an issue and we can wait for some volunteers
